platform/darwin/src/MGLAttributedExpression.h

```diff
@@ -64,17 +64,17 @@ MGL_EXPORT
 /** 
  Returns an `MGLAttributedExpression` object initialized with an expression and text format attributes.
  */
-- (instancetype)initWithExpression:(NSExpression *)expression attributes:(nonnull NSDictionary <MGLAttributedExpressionKey, id> *)attrs;
+- (instancetype)initWithExpression:(NSExpression *)expression attributes:(nonnull NSDictionary <MGLAttributedExpressionKey, NSExpression *> *)attrs;
 
 /**
  Creates an `MGLAttributedExpression` object initialized with an expression and the format attributes for font names and font size.
  */
-+ (instancetype)attributedExpression:(NSExpression *)expression fontNames:(nullable NSArray<NSString*> *)fontNames fontSize:(nullable NSNumber *)fontSize;
++ (instancetype)attributedExpression:(NSExpression *)expression fontNames:(nullable NSArray<NSString*> *)fontNames fontScale:(nullable NSNumber *)fontScale;
```

Comment:
This is kind of out of left field, but I think we could simplify the API and documentation considerably by removing parameters such as `fontNames` and `attributes` and the `attributes` property, in favor of explicit properties like `fontNames`. After all, unlike NSAttributedString, we don’t support storing arbitrary attributes on ranges of text.

Reply:
This particular initializer won’t scale well as we add more attributes over time. So maybe we should make the developer configure the MGLAttributedExpression over multiple statements, by setting the (read-write) properties individually.

Reply:
Per our slack conversation it is convenient to have an dictionary to create an object in one statement.